src/trio/_threads.py

```diff
@@ -206,9 +209,8 @@ def in_trio_thread() -> None:
 
 
 @attrs.frozen(eq=False, slots=False)
-# Explicit .../"Any" is not allowed
-class RunSync(Generic[RetT]):  # type: ignore[misc]
-    fn: Callable[..., RetT]  # type: ignore[misc]
+class RunSync(Generic[RetT]):  # type: ignore[explicit-any]
+    fn: Callable[..., RetT]  # type: ignore[explicit-any]
```

Comment:
can you use `Unpack[Ts]` here too?

Reply:
No. `Unpack` is only valid in function definitions and specifically requires it to be in a variadic argument.